.github/actions/e2e/setup-cluster/action.yaml

```
@@ -82,6 +82,11 @@ runs:
       PRIVATE_CLUSTER: ${{ inputs.private_cluster }}
       GIT_REF:  ${{ inputs.git_ref }}
     run: |
+      if [[ "$GIT_REF" == '' ]]; then 
+        REF=$(git rev-parse HEAD)
+      else 
+        REF=$GIT_REF
+      fi 
```

Why can't we just use the same variable? 

```suggestion
      if [[ "$GIT_REF" == '' ]]; then 
        GIT_REF=$(git rev-parse HEAD)

```